src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,233 @@
+[
+  {
+    "name":"io.netty.util.internal.shaded.org.jctools.queues.BaseMpscLinkedArrayQueueColdProducerFields",
```

Comment:
These are netty-related bits. We should remove these and leave configuration up to netty.

Reply:
👍 feel free to open an issue there